### PR TITLE
Add separate flag to disable compute size changes

### PR DESCRIPTION
**Pull request by joshenlim**

### CODE REVIEW

File: apps/studio/components/interfaces/Settings/Addons/Addons.tsx [hunk 1]
```diff
@@ -249,14 +250,15 @@ const Addons = () => {
                       </p>
                     )}
                     <ProjectUpdateDisabledTooltip
-                      projectUpdateDisabled={projectUpdateDisabled}
+                      projectUpdateDisabled={computeSizeChangesDisabled}
```
Comment (alaister) [inline apps/studio/components/interfaces/Settings/Addons/Addons.tsx:253]: We don't want to disable this when `projectUpdateDisabled` is enabled as well?
```suggestion
                      projectUpdateDisabled={computeSizeChangesDisabled || projectUpdateDisabled}
```

Comment (joshenlim) [reply apps/studio/components/interfaces/Settings/Addons/Addons.tsx:253]: so disabling compute sizes will be its own flag, separate from the flag that controls "project creation and update"

Comment (joshenlim) [reply apps/studio/components/interfaces/Settings/Addons/Addons.tsx:253]: hmm actually good point...

Comment (joshenlim) [reply apps/studio/components/interfaces/Settings/Addons/Addons.tsx:253]: yup okay i'll add that in! thanks man! 🙏 🙏

**Review by alaister — APPROVED** [review]
LGTM!

**Pull request merged**